src/dexcandles-v2.ts

```diff
@@ -37,62 +52,67 @@ export function handleSwapV2(event: SwapV2): void {
       candle = new Candle(candleId);
```

Comment:
The candleId should also be affected by the sort as in:

```
const candleId = periodStart
      .toString()
      .concat(candlestickPeriods[i].toString())
      .concat(token0.id)
      .concat(token1.id);
```
`token0` & `token1`

Reply:
Addressed